mlserver/registry.py

```diff
@@ -59,7 +50,7 @@ def __init__(
         on_model_reload: List[ModelReloadHook] = [],
         on_model_unload: List[ModelRegistryHook] = [],
     ):
-        self._versions: Dict[str, MLModel] = {}
+        self._versions: Dict[Union[str, None], MLModel] = {}
```

Comment:
Oh, I see what you did there! That's a pretty neat idea - makes for a way simpler implementation :+1: